3rdParty/LightPcapNg/LightPcapNg/src/light_pcapng_ext.c

```diff
@@ -35,8 +37,7 @@ struct _light_pcapng_t
 {
 	light_pcapng pcapng;
 	light_pcapng_file_info *file_info;
-	light_pcapng pcapng_iter;

```

Comment:
I'm not entirely sure why you removed `pcapng_iter`. I saw it was being used extensively in the code, and instead you now use the `pcapng` member. Is `pcapng_iter` really redundant?

Reply:
Yes it was used in the old code all over the place because everything was in memory and this was the pointer to the "next" block or something if I recall.

With the new code there is never more than a few blocks in memory (usually only 1) so this kind of bookkeeping isn't necessary. Everything you are working with is contained in `pcapng` so only use that variable. I think this is way more clear/concise.

I could have left it there, but I figured it would be more confusing for the next person working on the code to leave a dead member variable behind.